### PR TITLE
docstring fixes

### PR DESCRIPTION
**Pull request by RickSanchezStoic**
docstring fixes

### CODE REVIEW

**Comment by RickSanchezStoic** [comment]
I have modified the docstring tests to fail if it fails to match the docstring to the output or any runtime error occurs, unlike before when it used to pass anyway giving a summary of the failing functions. Hope this will ensure that any new formatting and/or any change to the functions that alter the output in any way will be checked immediately before being merged.

**Review by djl11 — APPROVED** [review]
lgtm!

File: ivy_tests/test_docstrings.py [hunk 1]
```diff
@@ -30,8 +33,28 @@ def test_docstrings(backend):
...
+        "num_cpu_cores",
+        "get_all_ivy_arrays_on_dev",
+        "num_ivy_arrays_on_dev",
+        "function_unsupported_dtypes",
+        "randint",
+        "unique_counts",
+        "unique_all",
     ]
 
+    skip_list_temp = [
```
Comment (simonetgordon) [inline ivy_tests/test_docstrings.py:46]: Why are these being skipped? Maybe you can add in a comment at the top of the function explaining why.

Comment (RickSanchezStoic) [reply ivy_tests/test_docstrings.py:46]: yeah, these are skipped because of an issue in their implementation, I will comment these , thanks!

**Comment by RickSanchezStoic** [comment]
Seems alright! I will merge now.

**Pull request merged**